### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by prateek1361**

### CODE REVIEW

File: cart-page/cart-page.css [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    .nav{
+     padding: 5px 10px;
+      border: 2px solid ;
+      border-radius: .5rem;
+      border: 2px solid black;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      background-color: #334366;
+      box-shadow: 0 8px 12px black;
```
Comment (Aditi0505) [inline cart-page/cart-page.css:15]: Use hex code for consistency.

File: cart-page/cart-page.css [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+      }
+      .grid-cards-icons:hover {
+        background: #334366;
+        color: whitesmoke;
+        opacity: 0.9;
+      }
+      .my-cart-sm {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
```
Comment (Aditi0505) [inline cart-page/cart-page.css:201]: Since this is repeating, you can create a common class for this.

File: cart-page/cart-page.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+      <div class="cart-grid">
+        <main class="main-cart">
+            <h3 class="cart-heading btm-b">My Cart - <span class="cart-desc"> 2 items</span></h3>
+            <ul class="cart-products">
+                <li class="cart-main">
+                    <div class="card cards-horizontal">
+                        <div class="img-container">
+                          <img
+                            src="../assets/cyberpunk.jpeg"
+                            alt="product image"
```
Comment (Aditi0505) [inline cart-page/cart-page.html:46]: Can come up with a better alt description - e.g Cyberpunk-product etc.

File: cart-page/cart-page.html [hunk 4]
```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GAMEWORLD</title>
```
Comment (TheAmlanSaikia) [inline cart-page/cart-page.html:7]: You can include author meta tags

**Review by TheAmlanSaikia — APPROVED** [review]
LGTM.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. You can cross-check across multiple places. Rest looks good to me.

**Pull request merged**